modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -1,14 +1,49 @@
-[id="anchor-and-file-names"]
-= Anchor Names and File Names
+[id="module_anchor-and-file-names-concept"]
+= File Names and Anchors
 
-To optimize modular documentation, follow these guidelines for naming module anchors and files:
+To optimize modular documentation, follow these guidelines for naming files and creating anchors:
 
-Anchor names:: Provide an anchor in the format `+++[id="anchor-name_{context}"]+++` for every module so that it can be identified by Asciidoctor when reused or cross-referenced. `+++{context}+++` is a variable whose value you define in the assembly. Give the anchor the same or similar name as the module heading. Separate the words in the anchor with hyphens and use an underscore to separate the `+++{context}+++` part:
-+
+.File names
+Assembly and module file names should accurately and closely reflect the title of the assembly or module. File names should have the format `prefix-filename.adoc` or `prefix_filename.adoc` where `prefix` is one of the following module prefixes:
```

Comment:
Why or? The prefix should be joined to the filename using the underscore, right?

prefix_<filename>.adoc

Further, we should remove 'should' from here as this is only a recommendation, and not a hard requirement as teams can choose not to do this and go the attribute route.

Reply:
Both hyphens and underscores are valid prefix separators. 'Should' implies that these are recommendations.

Reply:
I disagree that "should" implies that it's optional. It implies that it's required. I suggest following the IBM Style Guide's guidance here:

> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.

Reply:
+1 @kalexand-rh 

Also, since the sub-section about anchors addresses the reader directly ("At the top of every module, provide an anchor in the.."), perhaps this section can also be written that way - doing so will also avoid the use of "should".

Reply:
Good suggestion! I've rewritten and removed all 'shoulds' from the section.